kong/db/schema/entities/routes.lua

```diff
@@ -34,6 +34,11 @@ local entity_checks = {
   }},
 }
 
+local snis_elements_type = typedefs.wildcard_host
+
+if kong_router_flavor == "traditional" then
```

Comment:
Could we use a simple `kong_router_flavor == "traditional" and ... or ...`?

Reply:
I found in some test environments, the `kong.configuration.router_flavor` doesn't have a value so I set the default value `snis_elements_type` to `typedefs.wildcard_host`